tools/lsp/ui/main.slint

```diff
@@ -67,17 +67,13 @@ export component PreviewUi inherits Window {
                         Api.style-changed();
                     }
 
-                    Button {
-                        icon: Icons.sync;
-                        colorize-icon: true;
-                        clicked => { Api.reload-preview(); }
-                    }
                     Button {
                         icon: Icons.inspect;
                         colorize-icon: true;
                         checkable: true;
                         checked <=> preview.select-mode;
                         enabled: !preview.design-mode;
+                        visible:!preview.design-mode; 
```

Comment:
Sorry to be so naggy, but we do not have a formatter yet...

Please add a space after the `:`.